Doc/extending/extending.rst

```diff
@@ -203,31 +203,42 @@ function usually raises :c:data:`PyExc_TypeError`.  If you have an argument whos
 value must be in a particular range or must satisfy other conditions,
 :c:data:`PyExc_ValueError` is appropriate.
 
-You can also define a new exception that is unique to your module. For this, you
-usually declare a static object variable at the beginning of your file::
+You can also define a new exception that is unique to your module.
+For this, you can declare a static global object variable at the beginning
+of the file::
```

Comment:
This is **not correct** for multi-phase initialization, without preventing the `exec` function from other module instances from overriding this pointer.

Reply:
@encukou I added but then reverted (https://github.com/python/cpython/commit/6e5b1c59c793eef7d041a531155a0aa9d0ca9018) module state for this error type -- should we un-revert that commit? Sorry for my mistake here.

A

Reply:
I think it's fine -- module state deserves a better explanation a bit later in the tutorial. And the `ImportError` being added in the next PR is a good thing to show.

The main thing I'd do differently is not backporting to 3.13 -- that's like pushing directly to production. 3.14 is out in a few months; that sounds like a good amount of time for this all to spend in prerelease.